app/models/cdx/client.rb

```diff
@@ -7,25 +7,27 @@ def initialize(opts={})
 
   def default_opts
     {
-      :wsdl => "https://devngn.epacdxnode.net/cdx-register/services/RegisterSignService?wsdl",
-      :pretty_print_xml => true,
-      :log => true,
-      :soap_version => 2,
-      :convert_request_keys_to => :none,
+      wsdl: "https://devngn.epacdxnode.net/cdx-register/services/RegisterSignService?wsdl",
```

Comment:
added URL base to env var (with default value) here: eec1fa4

What do you think of that? I assume the rest of the URL will remain the same (but that is an assumption)